modules/load_waypoint_name_to_coordinates_map.py

```diff
@@ -3,9 +3,11 @@
 """
 import pathlib
 
+from modules.waypoint import Waypoint
```

Comment:
Use relative import and import namespace rather than class itself: `from . import waypoint`